event_seats_per_registration/models.py

```diff
@@ -0,0 +1,81 @@
+# -*- coding: utf-8 -*-
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+# © 2015 Grupo ESOC Ingeniería de Servicios, S.L.U.
+
+from openerp import _, api, fields, models
+from . import exceptions
+
+
+class Event(models.Model):
+    """Events enhaced with participants per registration limits."""
+
+    _inherit = "event.event"
+
+    seats_per_registration_max = fields.Integer(
```

Comment:
Use a shorter technical name: `max_seats`. It will ease any code logic. You can put in the label a longer description